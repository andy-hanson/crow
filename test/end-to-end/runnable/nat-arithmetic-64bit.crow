import
	crow/io/print: out, print

main void() summon, unsafe
	out print "{0x0d15ea5e0d15ea5e::nat64 & ~7 to-hex}"
