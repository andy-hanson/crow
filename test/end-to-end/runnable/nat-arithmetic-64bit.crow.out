d15ea5e0d15ea58
